communitycall-data.yml

```diff
@@ -46,4 +46,4 @@
     "2024-11-07": "mu"
     "2024-11-28": "fz"
     "2024-12-12": "js"
-    "2024-12-19": "kg"
\ No newline at end of file
+    "2024-12-19": "kg"
```

Comment:
This change looks spurious to me. Something like \no newline at the end of file?

Reply:
Yes